docs/howto/configure/auth-management.md

```diff
@@ -38,19 +40,113 @@ So we want to manage authentication by:
    config:
      jupyterhub:
        auth:
-         # will be renamed allowedlist in future JupyterHub
-         whitelist:
-           users:
+         allowed_users:
              # WARNING: THESE USER LISTS MUST MATCH (for now)
              - user1@gmail.com
              - user2@gmail.com
-         admin:
+         admin_users:
            users:
              # WARNING: THESE USER LISTS MUST MATCH (for now)
              - user1@gmail.com
              - user2@gmail.com
    ```
 
 ```{admonition} Switching auth
-Switching authentication for a pre-existing hub will simply create new usernames. Any pre-existing users will no longer be able to access their accounts (although administrators will be able to do so). If you have pre-existing users and want to switch the hub authentication, rename the users to the new auth pattern (e.g. convert github handles to emails).
-```
\ No newline at end of file
+Switching authentication providers (e.g. from GitHub to Google) for a pre-existing hub will simply create new usernames. Any pre-existing users will no longer be able to access their accounts (although administrators will be able to do so). If you have pre-existing users and want to switch the hub authentication, rename the users to the new auth pattern (e.g. convert github handles to emails).
+```
+
+## Native JupyterHub OAuthenticator for GitHub Orgs and Teams
+
+```{admonition}
+This setup is currently only supported for communities that **require** authentication via a GitHub organisation or team.
+
+We may update this policy in the future.
+```
+
+For communities that require authenticating users against a GitHub organisation or team, we instead use the native JupyterHub OAuthenticator.
+Presently, this involves a few more manual steps than the `auth0` setup described above.
+
+1. **Create a GitHub OAuth App.**
+   This can be achieved by following [GitHub's documentation](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
+   Use the "Switch account" button at the top of your settings page to make sure you have `2i2c-org` selected.
+   That way, the app will be owned by the `2i2c-org` GitHub org, rather than your personal GitHub account.
```

Comment:
```suggestion
   - Use the "Switch account" button at the top of your settings page to make sure you have `2i2c-org` selected.
     That way, the app will be owned by the `2i2c-org` GitHub org, rather than your personal GitHub account.
```